FHEM/00_SIGNALduino.pm

```diff
@@ -1051,13 +1051,16 @@ sub SIGNALduino_Get_Command {
 }
 
 ############################# package main
-sub SIGNALduino_Get_Command_CCReg {
+sub SIGNALduino_Get_Command_CCReg{
   my ($hash, @a) = @_;
+  return 'not enough number of arguments' if $#a < 1;
+  return 'Wrong command provided' if $a[0] != 'ccreg';
+  
   my $name=$hash->{NAME};
-  if (exists($cc1101_register{uc($_[2])}) || $_[2] =~ /^99$/ ) {
+  if (exists($cc1101_register{uc($a[1])}) || $a[1] =~ /^99$/ ) {
```

Comment:
Die hier meinte ich. Aber sorry, ich habe mich vertan. Ich hatte bei mir nur die 2 in eine 1 geändert. Hat sich erledigt.